src/CurrentTime.java

```diff
@@ -0,0 +1,12 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class CurrentTime {
+
+    public static String getCurrentTime() {
+        LocalDateTime now = LocalDateTime.now();
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy-HH-mm");
```

Comment:
Instead of hardcoding this pattern `"dd.MM.yyyy-HH-mm"` you can pass `String pattern `as an argument to this method.